bears/general/FilenameBear.py

```diff
@@ -38,6 +46,21 @@ def run(self, filename, file,
         """
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
+
+        if file_naming_convention is None:
+            self.warn('Please specify a file naming convention explicitly. '
+                      'Using the default "snake" naming convention...')
+            file_naming_convention = 'snake'
```

Comment:
yeah though we should maybe use `auto` still, so the user directly gets the language-default. When you supply Java files and it defaults to snake you think "huh what the... why snake?".